src/server.js

```diff
@@ -202,6 +210,48 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_EVENT_ID}/restaurants`,
+  getEvent,
+  async (request, response) => {
+    const { event } = request;
+    const users = event.users || {};
+    const userInput = Object.values(users);
+
+    if (userInput.length > 0) {
+      const lat = userInput[0].lat.toString();
```

Comment:
would you want to get the lat/lng from the datastore instead? is there a lat/lng attached to this request?

Reply:
This gets the Lat and Long from datastore using the getEvent function. Is that what you mean?

Reply:
@sooniln Any request that passes through the `getEvent` middleware triggers a call to datastore that attaches the correct event object to the body of the request.

Reply:
ah, I see, I missed the getEvent call in the initial arguments haha. getting confused by the JS from time to time!